planet/planetsympy/config

```diff
@@ -365,3 +365,9 @@ feed 15m http://sartajsingh.me/feeds/sympy.xml
         define_face hackergotchi/leosartaj.jpg
         define_facewidth 80
         define_faceheight 80
+
+feed 15m http://s-hacker.info/blog/gsoc16/feeds/sympy.xml
```

Comment:
Usually most of the blog platforms create feeds automatically, If you are using something like jekyll to create static sites, you can automate the process by creating a structure like this"

```
---
layout: null
---

<?xml version="1.0" encoding="utf-8"?>
<feed xmlns="http://www.w3.org/2005/Atom">

 <title>{{ site.title }}</title>
 <link href="{{ site.url }}/blog/feeds/sympy.xml" rel="self"/>
 <link href="{{ site.url }}/blog"/>
 <updated>{{ site.time | date_to_xmlschema }}</updated>
 <id>{{ site.url }}</id>
 <author>
   <name>{{ site.author.name }}</name>
   <email>{{ site.author.email }}</email>
 </author>

 {% for post in site.tags.SymPy %} <!-- Identifier for SymPy related Posts>
 <entry>
   <title>{{ post.title }}</title>
   <link href="{{ site.url }}{{ post.url }}"/>
   <updated>{{ post.date | date_to_xmlschema }}</updated>
   <id>{{ site.url }}{{ post.id }}</id>
   <content type="html">{{ post.content | xml_escape }}</content>
 </entry>
 {% endfor %}

</feed>
```

or if you do it manually, you might want to have a look at: https://github.com/lkiesow/python-feedgen